serverlistplayersfix.cpp

```
@@ -99,20 +100,35 @@ void ServerListPlayersFix::UpdatePlayers()
 {
 	auto gpGlobals = engine->GetServerGlobals();
 	g_pEntitySystem = GameEntitySystem();
+	auto steamGameServer = g_steamAPI.SteamGameServer();
 
-	if(!gpGlobals || !g_pEntitySystem)
+	if(!gpGlobals || !g_pEntitySystem || !steamGameServer)
```

Clang (14.0.6)

[warning] statement should be inside braces

(readability-braces-around-statements)